packages/netlify-cms-core/src/reducers/collections.js

```diff
@@ -128,7 +128,7 @@ export const selectInferedField = (collection, fieldName) => {
   const fields = collection.get('fields');
   let field;
 
-  // If colllection has no fields or fieldName is not defined within inferables list, return null
+  // If collection has no fields or fieldName is not defined within inferables list, return null
```

Comment:
I must have looked at this in annoyance a thousand times but never bothered to fix lol 👍👍